pakwheels_clone/cars/models.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib import admin
+from django.db import models
+
+
+from users.models import User
+
+
+class CarManufacturer(models.Model):
```

Comment:
prefixing every model with app name or some core concept isn't necessary as you are already in an app named the same thing